### ISSUE TITLE
Unable to control old Leviton switches (VRS05/VRS15)

### ISSUE THREAD

**Issue by denix0** [zwave-js/zwave-js-ui#3168]
I have several old Leviton switches (and dimmers) around the house - those are of the first generation of Z-Wave from about 10-15 years ago and I've been using them w/o problems with several different controllers and hubs throughout the years.

I'm setting up everything now in Z-Wave JS UI and noticed that I cannot control them either directly from the UI or from HomeAssistant via WebSocket. Moreover, in the UI under "Binary Switch v0" section there's "Current value" field, but there's NO "Target value" at all, which I normally get for other devices with e.g. "Binary Switch v1":

![binary-switch-v0](https://github.com/zwave-js/zwave-js-ui/assets/22880/e7bfdcf1-d7c9-432c-bb94-400cc02848c4)
![binary-switch-v1](https://github.com/zwave-js/zwave-js-ui/assets/22880/56a2017e-0a24-4747-9c1e-73ea13632fb8)

I'm using Zooz 800-series ZST39 LR stick. The problematic devices are Leviton VRS05 and VRS15 switches, but I haven't tried dimmers yet.

They report Z-Wave protocol version 2.9 and Z-Wave chip firmware version "0.4" or "0.5"

I wonder if this is due to "v0" in the "Binary Switch" class:

![leviton-vrs15-values](https://github.com/zwave-js/zwave-js-ui/assets/22880/566c6be0-4856-4a1f-bd78-b63e68ad12f0)

Or if it's a driver issue...

I would appreciate any help and gladly provide logs or try different options. Thanks in advance!

**Comment by robertsLando** [comment]
Zwave-js-ui version?

Also make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

Possibly with a test write of the target value

**Comment by denix0** [comment]
Yes, everything is latest, but the issue has been going on for a while, so I already went through several versions.

zwave-js-ui: 8.19.0.487524b
zwave-js: 11.0.0

[zwave-js-ui-store.zip](https://github.com/zwave-js/zwave-js-ui/files/12053742/zwave-js-ui-store.zip)

How can I force a test write if there's no control? I tried toggling the switch from HA, but not sure if that worked, as it always comes back to the current value.

**Comment by denix0** [comment]
Hi, any more suggestions I can try to help debug this issue? Thanks a lot!
